src/herder/HerderSCPDriver.h

```diff
@@ -32,6 +33,11 @@ class VirtualTimer;
 struct StellarValue;
 struct SCPEnvelope;
 
+// First protocol version supporting the application-specific weight function
+// for SCP leader election.
+ProtocolVersion constexpr APPLICATION_SPECIFIC_WEIGHT_PROTOCOL_VERSION =
```

Comment:
nit: could we name this parameter such that it's clear we're talking about nomination protocol and leader election?

Reply:
Renamed!